src/platform/packages/private/kbn-esql-editor/src/editor_footer/errors_warnings_popover.tsx

```diff
@@ -81,6 +81,7 @@ function ErrorsWarningsContent({
                                 &:hover {
                                   cursor: pointer;
                                 }
+                                white-space: pre-line;
```

Comment:
This is intended to fix this problem

<img width="1256" height="1678" alt="Image" src="https://github.com/user-attachments/assets/863b9a03-2251-4acb-9189-6205ec9bbffd" />